modules/statics/src/coins.ts

```diff
@@ -1121,4 +1123,36 @@ export const coins = CoinMap.fromCoins([
   teosToken('teos:IQ', 'Everipedia', 3, 'testtoken112', UnderlyingAsset.IQ, AccountCoin.DEFAULT_FEATURES, '', 'IQ'),
   teosToken('teos:BOX', 'Box', 6, 'kvszn1xyz1bu', UnderlyingAsset.EOS_BOX, AccountCoin.DEFAULT_FEATURES, '', 'BOX'),
   teosToken('teos:USDT', 'Tether', 4, 'lionteste212', UnderlyingAsset.USDT, AccountCoin.DEFAULT_FEATURES, '', 'USDT'),
+  avaxErc20('QI', 'BenQi', 18, '0x8729438eb15e2c8b576fcc6aecda6a148776c0f5', UnderlyingAsset['avaxc:QI']),
```

Comment:
@jrafael-simtlix should the first parameter be the underlying asset name for all tokens? For example should it be this? 

```suggestion
  avaxErc20('avaxc:QI', 'BenQi', 18, '0x8729438eb15e2c8b576fcc6aecda6a148776c0f5', UnderlyingAsset['avaxc:QI']),
```

Reply:
Done